tests/Utilities_test.cpp

```
@@ -328,6 +328,126 @@ SCENARIO("Reading and writing Delaunay triangulations to files" *
                  utilities::detail::canonical_topology_fingerprint(annotated));
       }
     }
+    WHEN("Distinct causal vertices occupy the same geometric point")
+    {
+      TemporaryDirectory const directory;
+      auto const               filename  = directory.file("coincident.off");
+      auto                     annotated = manifold.delaunay_snapshot();
```

Cppcheck (2.21.0)

[style] The function 'print_details' is never used.

(unusedFunction)